pkg/apis/apps/v1beta2/conversion.go

```diff
@@ -54,6 +55,12 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 		Convert_extensions_DeploymentStrategy_To_v1beta2_DeploymentStrategy,
 		Convert_v1beta2_RollingUpdateDeployment_To_extensions_RollingUpdateDeployment,
 		Convert_extensions_RollingUpdateDeployment_To_v1beta2_RollingUpdateDeployment,
+		Convert_extensions_DaemonSet_To_v1beta2_DaemonSet,
+		Convert_v1beta2_DaemonSet_To_extensions_DaemonSet,
+		Convert_extensions_DaemonSetSpec_To_v1beta2_DaemonSetSpec,
+		Convert_v1beta2_DaemonSetSpec_To_extensions_DaemonSetSpec,
+		Convert_extensions_DaemonSetUpdateStrategy_To_v1beta2_DaemonSetUpdateStrategy,
+		Convert_v1beta2_DaemonSetUpdateStrategy_To_extensions_DaemonSetUpdateStrategy,
```

Comment:
Would you move these closer to other DaemonSet conversion codes? The TODO comment does not apply to these conversion codes.